app/process/handler/message/handler.go

```diff
@@ -47,12 +53,16 @@ func NewHandler(
 		log.Fatalf("Invalid topic id: [%v]", topicId)
 	}
 
+	participationRate := prometheusServices.NewGaugeMetric("validators_participation_rate", "Participation rate: Track validators' activity in %.")
```

Comment:
Can we extract this metric information either in a config file or in a constants file that will store all future metrics

Reply:
done with constants